### ISSUE TITLE
CrashLoopBackOff Kube-controller-manager-minikube

### ISSUE THREAD

**Issue by Firebass** [kubernetes/minikube#4047]
Operating System Debian 9.0 Stretch
In VMware Workstation 14 

I tried many times to fix the problem but somehow i don't get it to work.
After typing minikube start it shows --> Temporary Error: Error getting service kubernetes-dashboard: services "kubernetes-dashboard" not found
and when i type kubectl get po --all-namespaces -->
NAMESPACE     NAME                               READY   STATUS             RESTARTS   AGE
kube-system   etcd-minikube                      1/1     Running            1          101m
kube-system   kube-addon-manager-minikube        1/1     Running            1          101m
kube-system   kube-apiserver-minikube            1/1     Running            1          101m
kube-system   kube-controller-manager-minikube   0/1     CrashLoopBackOff   25         102m
kube-system   kube-scheduler-minikube            1/1     Running            1          101m

can someone help me please?

**Comment by Firebass** [comment]
![grafik](https://user-images.githubusercontent.com/49242474/55487923-5752b300-562f-11e9-9145-3d38fe2680b6.png)

**Comment by tstromberg** [comment]
Do you mind attaching the output of `minikube logs`? Thanks!

**Comment by tstromberg** [comment]
It's also possible that `minikube delete` may resolve your problem. In any case, I'd also like to see the exact `minikube start` command-line used.

**Comment by Bender250** [comment]
I am seeing the same issue. Deleting `minikube` helps for some time, but then it again starts restarting.

```
> minikube start --memory 8192 --cpus 4
😄  minikube v1.0.0 on linux (amd64)
🤹  Downloading Kubernetes v1.14.0 images in the background ...
🔥  Creating virtualbox VM (CPUs=4, Memory=8192MB, Disk=20000MB) ...
📶  "minikube" IP address is 192.168.99.104
🐳  Configuring Docker as the container runtime ...
🐳  Version of container runtime is 18.06.2-ce
⌛  Waiting for image downloads to complete ...
✨  Preparing Kubernetes environment ...
🚜  Pulling images required by Kubernetes v1.14.0 ...
🚀  Launching Kubernetes v1.14.0 using kubeadm ... 
⌛  Waiting for pods: apiserver proxy etcd scheduler controller dns
🔑  Configuring cluster permissions ...
🤔  Verifying component health .....
💗  kubectl is now configured to use "minikube"
🏄  Done! Thank you for using minikube!
```

# System info
`> uname -a`
`Linux infsec-server 4.9.0-8-amd64 #1 SMP Debian 4.9.144-3.1 (2019-02-19) x86_64 GNU/Linux`

`> VirtualBoxVM -h`
`Oracle VM VirtualBox VM Runner v6.0.4`

[kube-scheduler-minikube.txt](https://github.com/kubernetes/minikube/files/3085069/kube-scheduler-minikube.txt)
[kube-controller-manager-minikube.txt](https://github.com/kubernetes/minikube/files/3085070/kube-controller-manager-minikube.txt)
[minikube_logs.log](https://github.com/kubernetes/minikube/files/3085071/minikube_logs.log)

**Comment by Firebass** [comment]
I have manage to solve the Probleme bei reinstalling Minikube. Thank you very much!  
 
  Am Fr., Apr. 5, 2019 at 2:22 schrieb Thomas Strömberg<notifications@github.com>:   
Do you mind attaching the output of minikube logs? Thanks!

**Comment by Bender250** [comment]
> I have manage to solve the Probleme bei reinstalling Minikube. 

What do you mean by reinstalling? I am using latest release binary (`sha256sum a315869f81aae782ecc6ff2a6de4d0ab3a17ca1840d1d8e6eea050a8dd05907f`), so it makes no difference to remove and download it again.

Else I am not limited by this issue, I can just restart it periodically, as I am not using K8s in production. I just wanted to provide more information, if it can help the issue ;).

**Comment by tstromberg** [comment]
I believe this issue was resolved in the v1.1.0 release. Please try upgrading to the latest release of minikube and run `minikube delete` to remove the previous cluster state.

If the same issue occurs, please re-open this bug. Thank you opening this bug report, and for your patience!

**Comment by Bender250** [comment]
Thanks a lot! I have not experienced the issue since the update :).
